pkg/network/vmispec/interface.go

```diff
@@ -39,6 +39,16 @@ func FilterSRIOVInterfaces(ifaces []v1.Interface) []v1.Interface {
 	return sriovIfaces
 }
 
+func FilterInterfacesSpec(ifaces []v1.Interface, predicate func(i v1.Interface) bool) []v1.Interface {
```

Comment:
unused ?

Reply:
This should be moved to the `network, netstat: Reorder and clean status update steps` commit.
Done